torchvision/prototype/transforms/_container.py

```diff
@@ -25,9 +27,13 @@ def extra_repr(self) -> str:
         return "\n".join(format_string)
 
 
-class RandomApply(Compose):
-    def __init__(self, transforms: Sequence[Callable], p: float = 0.5) -> None:
-        super().__init__(transforms)
+class RandomApply(Transform):
```

Comment:
We inherited from `Compose` before, because both shared the same constructor as well as `extra_repr`. With this patch, we the constructor is different and thus I decided to just copy paste `extra_repr`.